src/fetch_package.js

```diff
@@ -122,12 +134,12 @@ export default async function fetchPackage(
         /**
          * We're caching.
          *
-         * Set the expiration of the cache at 900 seconds (15 minutes).
-         * This feels reasonable for now.
+         * Set the expiration of the cache to be really high (24 hours) as the
+         * files aren't expected to ever change.
          */
         return fetcher
             .use(superagentCache)
-            .expiration(900)
+            .expiration(24 * 60 * 60)
```

Comment:
SUPER NIT: I like the comments, and this is likely my fault since I wrote the original bits, but I wonder if an inline comment on this line of just `// 24 hours` would be useful?